sidebarDapp.ts

```diff
@@ -16,6 +16,7 @@ export const sidebarDapp: SidebarsConfig = {
       },
       items: [
         'dapp/sapphire/quickstart',
+        'dapp/sapphire/chain',
```

Comment:
```suggestion
        'dapp/sapphire/network',
```

Maybe this is more clear than original title.

Reply:
yes, I gave it the id network in the https://github.com/oasisprotocol/sapphire-paratime/pull/400